docs/examples/card/CardGrid/README.md

```diff
@@ -1 +1 @@
-undefined
\ No newline at end of file
+This component organizes a grid of cards that should be flushed left when there are incompelete rows.
```

Comment:
Not should be in my opinion -- but rather: ... a grid of cards that **are** flushed left when a row contains less than its full capacity.